src/nutrients_plankton_bacteria_detritus.jl

```diff
@@ -10,66 +10,78 @@ const c = Center()
 """
     NutrientsPlanktonBacteriaDetritus(; kw...)
```

Comment:
Should we call this something new? Either: NutrientsPlanktonBacteriaDetritusZoo OR just NutrientsPlanktonDetritus since phytoplankton and zooplankton and bacteria are all "plankton"?

Reply:
Talked to Hengdi: Let's leave this as is for now and have a larger conversation later about structure/modularity.

Reply:
Sounds good!